diff -up kdevelop-23.04.2/plugins/astyle/astyle_stringiterator.cpp.omv~ kdevelop-23.04.2/plugins/astyle/astyle_stringiterator.cpp
--- kdevelop-23.04.2/plugins/astyle/astyle_stringiterator.cpp.omv~	2023-06-10 01:21:39.475168014 +0200
+++ kdevelop-23.04.2/plugins/astyle/astyle_stringiterator.cpp	2023-06-10 01:21:47.284344366 +0200
@@ -22,7 +22,7 @@ AStyleStringIterator::~AStyleStringItera
 {
 }
 
-astyle::streamoff AStyleStringIterator::tellg()
+std::streamoff AStyleStringIterator::tellg()
 {
   return m_is.pos();
 }
@@ -59,7 +59,7 @@ void AStyleStringIterator::peekReset()
     m_peekStart = -1; // invalid
 }
 
-astyle::streamoff AStyleStringIterator::getPeekStart() const
+std::streamoff AStyleStringIterator::getPeekStart() const
 {
     // NOTE: we're not entirely sure if this is the correct implementation.
     // we're trying to work-around https://bugs.kde.org/show_bug.cgi?id=399048
diff -up kdevelop-23.04.2/plugins/astyle/astyle_stringiterator.h.omv~ kdevelop-23.04.2/plugins/astyle/astyle_stringiterator.h
--- kdevelop-23.04.2/plugins/astyle/astyle_stringiterator.h.omv~	2023-06-10 01:21:05.373399129 +0200
+++ kdevelop-23.04.2/plugins/astyle/astyle_stringiterator.h	2023-06-10 01:21:28.310916074 +0200
@@ -21,13 +21,13 @@ public:
     explicit AStyleStringIterator(const QString &string);
     ~AStyleStringIterator() override;
 
-    astyle::streamoff tellg() override;
+    std::streamoff tellg() override;
     int getStreamLength() const override;
     bool hasMoreLines() const override;
     std::string nextLine(bool emptyLineWasDeleted = false) override;
     std::string peekNextLine() override;
     void peekReset() override;
-    astyle::streamoff getPeekStart() const override;
+    std::streamoff getPeekStart() const override;
 private:
     QString m_content;
     QTextStream m_is;
